### PR TITLE
6.8 security

### PR DESCRIPTION
**Pull request by nelantone**

### CODE REVIEW

File: app/controllers/payments_controller.rb [hunk 1]
```diff
@@ -27,7 +27,6 @@ def create
       flash[:error] = "Unfortunately, there was an error processing your payment: #{err[:message]}"
     end
 
-    # @email = params[:email]
-    # UserMailer.create(@email, @product, charge ).deliver_now
+    UserMailer.create(@user, charge ).deliver_now
```
Comment (wazery) [inline app/controllers/payments_controller.rb:30]: Extra space here after `charge`.

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -14,11 +14,9 @@ def welcome(user)
...
-  # def create(email, product, charge)
-  #   @appname = "Bearlin Bikes"
-  #   mail( :to => email,
-  #         :subject => "Thank you for buying in #{@appname}! Here is your order") do |format|
-  #     format.html { render 'mail_order' }
-  #   end
-  # end
+  def create(user, charge)
+    @appname = "Bearlin Bikes"
+    mail( :to => user.email,
```
Comment (wazery) [inline app/mailers/user_mailer.rb]: Better to use new Ruby hash syntax like this:

`mail(to: user.email, subject: "Thank you for buying in #{@appname}! Here is your order")`

**Review by wazery — APPROVED** [review]
Good job so far, minor comments on the coding style.

**Comment by wazery** [comment]
Please also rebase the branch.

File: app/controllers/payments_controller.rb (hunk 1 above)
Comment (nelantone) [reply app/controllers/payments_controller.rb:30]: Hoh! yes

File: app/mailers/user_mailer.rb (hunk 2 above)
Comment (nelantone) [reply app/mailers/user_mailer.rb]: Yes! It's better like this :)

**Comment by nelantone** [comment]
Hoh! I forgot to rebase! I do it now :)

**Comment by nelantone** [comment]
I'm going to merge it directly to master and closing the branch *(big commits refactor)

**Pull request closed without merging**